### DISCUSSION TITLE
Since the profiles are being updated, are there plans for checking country statuses?

### DISCUSSION THREAD

**Discussion by Anymusxd** [ppy/osu#30541] [Q&A] [answered]
I'm asking because I've seen people (including myself) that their country flag didn't change after 6 months or more.

Before anything, yes, I've asked support to manually change my flag 5 months ago. At that time I was already 1 year moved out of the country so now it would be 1 year 5 months. My country flag still hasn't changed.

**Comment by SupDos** [comment]
Flags don't change automatically, you need to change them in your account settings after being in that country for a set amount of time https://osu.ppy.sh/home/news/2023-07-28-enabling-country-flag-changes

**Reply by Anymusxd** [reply]
Support told me to wait 6 months and I no have option for that on my profile/profile editor

**Reply by Anymusxd** [reply, marked as answer]
update:

It finally appeared hte option to change👍
